clang/lib/DPCT/Diagnostics.inc

```
@@ -227,3 +227,5 @@ DEF_WARNING(RESERVE_FOR_FUTURE, 1105, "")
 DEF_COMMENT(RESERVE_FOR_FUTURE, 1105, "")
 DEF_WARNING(EXTENSION_DEVICE_INFO, 1106, "'%0' was migrated with the Intel extensions for device information which may not be supported by all compilers or runtimes. You may need to adjust the code.")
 DEF_COMMENT(EXTENSION_DEVICE_INFO, 1106, "'{0}' was migrated with the Intel extensions for device information which may not be supported by all compilers or runtimes. You may need to adjust the code.")
+DEF_WARNING(OVERLOAD_UNSUPPORTED, 1107, "Unmigrated overload for %0.")
```

Why is this warning needed? Does it indicate a problem in SYCLomatic or in the user's code? I think that we should throw an exception if it's never expected or provide the user with a recommendation for resolving it.

---

Warning msg updated to be clear